clients/da-vinci-client/src/test/java/com/linkedin/davinci/store/rocksdb/RocksDBStoragePartitionTest.java

```diff
@@ -227,6 +229,143 @@ public void testIngestion(
         null,
         ROCKSDB_THROTTLER,
         rocksDBServerConfig);
+
+    // Test deletion
+    String toBeDeletedKey = KEY_PREFIX + 10;
+    Assert.assertNotNull(storagePartition.get(toBeDeletedKey.getBytes(), false));
+    storagePartition.delete(toBeDeletedKey.getBytes());
+    Assert.assertNull(storagePartition.get(toBeDeletedKey.getBytes(), false));
+
+    Options storeOptions = storagePartition.getOptions();
+    Assert.assertEquals(storeOptions.level0FileNumCompactionTrigger(), 40);
+    storagePartition.drop();
+    options.close();
+    removeDir(storeDir);
+  }
+
+  @Test(dataProvider = "True-and-False", dataProviderClass = DataProviderUtils.class)
+  public void testIngestionFormatVersionChange(boolean sorted) {
```

Comment:
Not sure about the impact to compaction because of this format change.
In theory, we could test the compact in the unit test.
For example, use different format versions to compose different L0 files and then manually run `RocksDB#compact`.

Reply:
I added couple of  `compactRang` calls in the test. Should be enough ?